commit-data/entrypoint.sh

```diff
@@ -2,6 +2,14 @@
 
 set -e -o pipefail
 
+if [[ $INPUT_WORKING_DIRECTORY ]]; then
+    cd $INPUT_WORKING_DIRECTORY
+    echo "Working directory is $INPUT_WORKING_DIRECTORY"
```

Comment:
You should echo first, in case the `cd` fails for some reason.

Reply:
Okay, makes sense.